docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-apps-java/cloud-ysql-yb-jdbc.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Build a Java application that uses YSQL
+headerTitle: Build a Java application
+linkTitle: Java
+description: Build a simple Java application using the YugabyteDB JDBC Driver and using the YSQL API to connect to and interact with a Yugabyte Cloud cluster.
+menu:
+  latest:
+    parent: cloud-build-apps
+    name: Java
+    identifier: cloud-java-1
+    weight: 550
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+The following tutorial shows a simple Java application that connects to a YugabyteDB cluster using the topology-aware [Yugabyte JDBC driver](../../../../../integrations/jdbc-driver/) and performs basic SQL operations. Use the application as a template to get started with Yugabyte Cloud in Java.
+
+## Prerequisites
+
```

Comment:
This section is left empty. Either remove or fill in. Probably the latter is necessary.